recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - 10.15                  # [osx and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - 10.15                  # [osx and x86_64]
```

Comment:
When I added this configuration, the errors decreased, before there were 20 errors ...